src/pages/InitMsig.vue

```diff
@@ -214,63 +223,77 @@ export default defineComponent({
 			errorMessage(this.key);
 			openErrorNotificationWithIcon(error.message);
 		},
+		createMultisig() {
+			let sender = algosdk.decodeUnsignedTransaction(
```

Comment:
I think we should rename it to fromAccount. The sender can be different. (the account that submitted the transaction)

Reply:
I did update the name in the second PR, actually both are inter-connected